Runtime/Services/BacktraceApi.cs

```diff
@@ -202,6 +208,11 @@ public IEnumerator Send(string json, List<string> attachments, Dictionary<string
             {
                 yield return request.SendWebRequest();
                 BacktraceResult result;
+#if UNITY_2020_2_OR_NEWER
+                var networkError = request.result == UnityWebRequest.Result.ConnectionError || request.result == UnityWebRequest.Result.ProtocolError;
```

Comment:
We might need to move `var networkError` outside of this #ifdef, otherwise the #else condition doesn't assign anything. Also consider renaming to `isNetworkError` for consistency